packages/core/src/modules/vc/W3cCredentialsModule.ts

```diff
@@ -1,4 +1,4 @@
-import type { W3cVcModuleConfigOptions } from './W3cCredentialsModuleConfig'
+import type { W3cCredentialsModuleConfigOptions } from './W3cCredentialsModuleConfig'
```

Comment:
Again, as this is unrelated with the qualified/unqualified dids thing just want to remark it to check if it's intended to add in this PR (no problem from my side ofc!).

Reply:
Yes, intended. Just came across it and thought let's fix it directly